src/main/java/org/symphonyoss/symphony/messageml/elements/TimezonePicker.java

```diff
@@ -0,0 +1,181 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.symphonyoss.symphony.messageml.MessageMLContext;
+import org.symphonyoss.symphony.messageml.MessageMLParser;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.form.TimezonePickerNode;
+import org.symphonyoss.symphony.messageml.util.XMLAttribute;
+import org.symphonyoss.symphony.messageml.util.XmlPrintStream;
+import org.w3c.dom.Node;
+
+import java.time.DateTimeException;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TimezonePicker extends FormElement implements LabelableElement, TooltipableElement {
```

Comment:
Would that be a good opportunity to put here the documentation from https://github.com/symphonyoss/messageml-utils/issues/246? (like we said with a bit of javadoc to help you understand what are the fields available in the element)

Reply:
Sure, i will add a javadoc to the class!